server/controllers/userAuth.js

```diff
@@ -411,6 +411,22 @@ const changePassword = (req, res) => {
         return doc.save();
       }
     })
+    .then(() => {
+      const {
+        sessionID,
+        sessionStore: store,
+        sessionStore: { db }
+      } = req;
+      const regexp = new RegExp(userId);
+
+      return db
+        .collection('sessions')
+        .find({
+          _id: { $ne: sessionID },
+          session: regexp
```

Comment:
will this query find userId in his session ?? wouldn't it destroy all the existing sessions in the database, even those ones that not belong to him??

cause session prop in DB looks like this 
`{"cookie":{"originalMaxAge":null,"expires":null,"httpOnly":true,"path":"/"},"passport":{"user":"5c9dc5a619ae7924074940f4"}}`

Reply:
We create a regexp with userId in line 420: `const regexp = new RegExp(userId)`. Mongo recognizes regexp and uses it to find this user id in the session's string. Other user's session remain.